.github/workflows/merge-main.yml

```diff
@@ -107,9 +106,24 @@ jobs:
           # Run certbot with one-off job
           oc create job "certbot-manual-$(date +%s)" --from=cronjob/certbot
 
+  image-promotions:
+    name: PROD Promotions
+    needs: [deploy-test]
```

Comment:
It'll keep moving the tags, but not change anything since no deploys are happening.  If this is undesirable we can gate that portion of the workflows too.  The upcoming imagestream consistency PR will make sure the cronjobs are using images from OpenShift too.